### PR TITLE
Fixing bootstrap config init

### PR DESCRIPTION
**Pull request by vedantjoshi84**
Related slack thread: https://platform9.slack.com/archives/C018NF9BF5X/p1688408280400309?thread_ts=1688402573.981169&cid=C018NF9BF5X

Verified that the nodelet cluster came up and the nodelet config file was properly filled up. 

Generated `nodelet-bootstrap-config.yaml`: 
```
[centos@vedant-smcp-nodelet conf]$ cat nodelet-bootstrap-config.yaml 

sshUser: centos
sshPrivateKeyFile: /home/centos/.ssh/id_rsa
nodeletPkg: /opt/pf9/airctl/nodelet/nodelet.tar.gz
clusterName: airctl-mgmt
k8sApiPort: "443"
masterIp: 10.128.145.254
mastervipvrouterid: 0
mtu: "1440"
privileged: "true"
containerruntime:
    name: containerd
    cgroupDriver: systemd
ipv4: true
calico:
    v4Interface: first-found
    v6Interface: first-found
    v4ContainersCidr: 10.20.0.0/22
    v6ContainersCidr: fd00:101::/116
    v4BlockSize: 26
    v6BlockSize: 122
    v4NATOutgoing: true
    v4IpIpMode: Always
masterNodes:
    - nodeName: 10.128.145.254
workerNodes: []
isAirgapped: false
```

### CODE REVIEW

File: nodeletctl/pkg/nodeletctl/nodeletctl.go [hunk 1]
```diff
@@ -15,72 +15,72 @@ import (
...
-	MasterVipEnabled       bool                   `json:"masterVipEnabled,omitempty" survey:"masterVipEnabled"`
-	MasterVipInterface     string                 `json:"masterVipInterface,omitempty" survey:"masterVipInterface"`
-	MasterVipVrouterId     int                    `json:"masterVipVrouterId,omitempty" survey:"masterVipVrouterId"`
-	MTU                    string                 `json:"mtu,omitempty" survey:"mtu"`
-	Privileged             string                 `json:"privileged,omitempty" survey:"privileged"`
+	SSHUser                string                 `json:"sshUser,omitempty" yaml:"sshUser,omitempty" survey:"sshUser"`
+	SSHPrivateKeyFile      string                 `json:"sshPrivateKeyFile,omitempty" yaml:"sshPrivateKeyFile,omitempty" survey:"sshPrivateKeyFile"`
+	CertsDir               string                 `json:"certsDir,omitempty" yaml:"certsDir,omitempty" survey:"certsDir"`
+	KubeConfig             string                 `json:"kubeconfig,omitempty" yaml:"kubeconfig,omitempty" survey:"kubeconfig"`
+	Pf9KubePkg             string                 `json:"nodeletPkg,omitempty" yaml:"nodeletPkg,omitempty" survey:"nodeletPkg"`
```
Comment (cruizen) [inline nodeletctl/pkg/nodeletctl/nodeletctl.go:22]: So the most important change is to rename `pf9KubePkg` to `nodeletPkg`

**Review by cruizen — APPROVED** [review]
Is the yaml part required?

File: nodeletctl/pkg/nodeletctl/nodeletctl.go (hunk 1 above)
Comment (vedantjoshi84) [reply nodeletctl/pkg/nodeletctl/nodeletctl.go:22]: No. The yaml part is important change.

**Review by mithilarun — APPROVED** [review]

File: nodeletctl/pkg/nodeletctl/nodeletctl.go (hunk 1 above)
Comment (cruizen) [reply nodeletctl/pkg/nodeletctl/nodeletctl.go:22]: Do we need to backport the YAML part to LTS1 too?

Comment (vedantjoshi84) [reply nodeletctl/pkg/nodeletctl/nodeletctl.go:22]: No I think, not needed. This is only for smcp.

**Pull request merged**